### ISSUE TITLE
Preview data read from the uploaded file

### ISSUE THREAD

**Issue by yehee** [eddieantonio/predictive-text-studio#277]
I noticed that we only have `edit` and `delete` options for the uploaded files, and when the user clicks `edit` icon, it automatically changes its type to `Direct Entry` although the user may not have an intention to edit. I think it would help users to navigate through the uploaded files and have a preview as in, what the application currently has for `edit` but not editable.

Would this be something we might be interested in working on? @eddieantonio

**Comment by eddieantonio** [comment]
Yes! This sounds like a good issue. I'll add it to the task board!
